.byebug_history

```diff
@@ -3,6 +3,100 @@ user
 continue
```

Comment:
why is this needed?

Reply:
?

Reply:
This one was automatically generated. I think it keeps track of debugger history.